devstats/config/devel/restore_db.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
What's this for?

Reply:
This is part of reinit script which is not in our fork